### PR TITLE
test(python-bdd): test IO to 2 Nexuses

### PR DESCRIPTION
**Pull request by jonathan-teh**
Create 2 nexuses each with 2 replicas, connect to both nexuses over
NVMf, check both paths are live, run fio against the block device, and
verify that IO is flowing to one path only.

Fixes CAS-974

### CODE REVIEW

**Comment by jonathan-teh** [comment]
bors try

File: test/python/test_bdd_nexus_multipath.py [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+    assert stat[1].stats.num_write_ops > 1000
+
+    stat = hdls["ms0"].stat_nvme_controllers()
+    assert stat[0].stats.num_write_ops == 0
+    assert stat[1].stats.num_write_ops == 0
+
+    uri = create_nexus
+    uri2 = create_nexus_2
+
+    nvme_disconnect(uri)
```
Comment (mtzaurus) [inline test/python/test_bdd_nexus_multipath.py:234]: Can we move cleanup code into 'create_nexus...' fixtures, straight before 'nexus_destroy()' ? That would assure cleanup step always takes place regardless of assertions in the test. For instance, If the assertion triggers at line 229, then 'nvme_disconnect()' is not reached.

Comment (jonathan-teh) [reply test/python/test_bdd_nexus_multipath.py]: Good idea, I've done this.

**Comment by jonathan-teh** [comment]
bors try

**Review by mtzaurus — APPROVED** [review]

**Review by gila — APPROVED** [review]

**Comment by jonathan-teh** [comment]
bors merge

**Pull request merged**